tests/unittests/delegated_operators_tests.py

```
@@ -807,21 +810,21 @@
-                sort_by=SortByField.QUEUED_AT,
+                sort_by=SortByField.SCHEDULED_AT,
                 sort_direction=SortDirection.ASCENDING,
             ),
         )
 
         self.assertEqual(len(docs), 1)
-        self.assertEqual(docs[0].id, queued[0].id)
+        self.assertEqual(docs[0].id, scheduled[0].id)
 
         docs = self.svc.list_operations(
             operator=f"@voxelfiftyone/operator/test_0",
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)